.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+exclude: '.git'
+default_stages: [commit]
+
+repos:
+  - repo: https://github.com/psf/black
```

Comment:
[black](https://github.com/psf/black) is probably the thing that is not-playing with Pylint at the moment. black will actually auto-format the code to pass the linter, rather than just flagging issues to change by hand.

It looks like they have minor style disagreements; Pylint can be made to cooperate by modifying the pylint config as described [here](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html).

[Here's a short thread](https://github.com/microsoft/presidio/issues/317) on a Microsoft project that was thinking about switching to black.